src/map/clif.c

```diff
@@ -1065,6 +1065,7 @@ void clif_set_unit_idle(struct block_list* bl, struct map_session_data *tsd, enu
 #endif
 #if PACKETVER >= 20150513
 	p.body = vd->body_style;
+	safestrncpy(p.name, status->get_name(bl), NAME_LENGTH);
```

Comment:
safestrncpy will crash too if one of parameters is null pointer.
Here possible null pointer in status->get_name(bl)